public/app/frontend/src/components/button/button.stories.js

```diff
@@ -0,0 +1,29 @@
+import button from './button.html.twig';
+import './index.js';
+
+export default {
+    title: 'Components/Button',
+};
+
+const Template = (args) => {
+    return button(args);
+};
+
+export const Primary = Template.bind({});
+Primary.args = {
+    buttonText: 'Search',
+    style: 'primary',
+}
```

Comment:
@EarthlingDavey the older docs use .bind https://storybook.js.org/tutorials/intro-to-storybook/ember/en/simple-component/

I think the recommended usage has changed between versions. I've used the Template and bind syntax as that's what I used last time I worked with Storybook but I'll have a look into the newer version as it might be better :)